<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePassword</class>
 <widget class="QDialog" name="ChangePassword">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>263</width>
    <height>149</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Change Password</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../../../qshutdown.qrc">
    <normaloff>:/red_glasses</normaloff>:/red_glasses</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="retypedNewL">
     <property name="text">
      <string>Retype password:</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="retypedNewPasswd">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="newPasswd">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="newL">
     <property name="text">
      <string>New password:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="oldL">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="text">
      <string>Old password:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="oldPasswd">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>oldPasswd</tabstop>
  <tabstop>newPasswd</tabstop>
  <tabstop>retypedNewPasswd</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="../../../qshutdown.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChangePassword</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ChangePassword</sender>
   <signal>rejected()</signal>
   <receiver>ChangePassword</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>131</x>
     <y>74</y>
    </hint>
    <hint type="destinationlabel">
     <x>131</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
